hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/UtilizationEndpoint.java

```diff
@@ -49,21 +55,24 @@ public class UtilizationEndpoint {
 
   private FileCountBySizeDao fileCountBySizeDao;
   private UtilizationSchemaDefinition utilizationSchemaDefinition;
+  private ContainerCountBySizeDao containerCountBySizeDao;
 
   @Inject
   public UtilizationEndpoint(FileCountBySizeDao fileCountBySizeDao,
+                             ContainerCountBySizeDao containerCountBySizeDao,
                              UtilizationSchemaDefinition
                                  utilizationSchemaDefinition) {
     this.utilizationSchemaDefinition = utilizationSchemaDefinition;
     this.fileCountBySizeDao = fileCountBySizeDao;
+    this.containerCountBySizeDao = containerCountBySizeDao;
   }
 
   /**
    * Return the file counts from Recon DB.
    * @return {@link Response}
    */
   @GET
-  @Path("/fileCount")
+  @Path("/pop")
```

Comment:
If this change is impacting taking care on UI, whether we have created separate task to handle UI changes ? Pls mention and make sure that PR for this change and PR for UI should be merged together.

Reply:
This was by mistake! thanks for pointing it out !!